### ISSUE TITLE
Feature request: Extend aoscx_facts for LLDP neighbors

### ISSUE THREAD

**Issue by smirlach** [aruba/aoscx-ansible-collection#79]
Hello,

i'd like to ask if it is possible to add a parameter to the aoscx_facts module to get all LLDP neighbors of a device similar to the CLI commands `show lldp neighbor-info`or `show lldp neighbor-info detail`?

I know the LLDP neighbors are also somehow included with the _physical_interfaces_ parameter, but it is no fun to iterate over all line cards and interfaces to get all of the LLDP neighbors. So this would be a very useful extension to me.

**Comment by tchiapuziowong** [comment]
Thank you for your feedback! I'll include this in our backlog and will update the issue once we have a planned development in place.

**Comment by tchiapuziowong** [comment]
@smirlach can you provide an example of how you'd like the data to be structured? A list, dictionary, list of dictionaries? An example like so would be greatly appreciated and will allow us to better plan/design this enhancement.

Example data:
```
[
{ "local_port": "1/1/1",
"chassis_id": "5c:b9:01:1d:00:00",
},
{ "local_port": "1/1/45",
"chassis_id": "90:20:c2:bb:9c:00",
},
]
```

**Comment by tchiapuziowong** [comment]
This has been included in the latest collection version v4.3.1 - closing issue

**Comment by tchiapuziowong** [comment]
@smirlach can you remove the `ansible_python_interpreter` variable from your playbook & inventory? since you're using a virtual environment that wouldn't be where your modules are installed - to validate where your `pyaoscx` package is installed execute `pip show pyaoscx`
